### DISCUSSION TITLE
How to get user's IP with v0.6?

### DISCUSSION THREAD

**Discussion by Extarys** [wobsoriano/trpc-nuxt#102] [Q&A]
I found a similar question here (https://github.com/wobsoriano/trpc-nuxt/issues/21) but it's now outdated as the `recipes` folder is in the v3 branch.

IP is useful when implementing auth or rate limiters.

Thanks

**Comment by wobsoriano** [comment]
Access it from the event context - https://trpc-nuxt.vercel.app/get-started/tips/authorization
